artifacts/scripts/util.sh

```diff
@@ -499,7 +499,10 @@ EOF
 # amend-godeps-at checks out the Godeps.json at the given commit and amend it to the previous commit.
 function amend-godeps-at() {
     if [ -f Godeps/Godeps.json ]; then
+        set +e
         git checkout ${f_mainline_commit} Godeps/Godeps.json # reset to mainline state which is guaranteed to be correct
```

Comment:
I still wonder how the Godeps.json can exist here, but is gone in `${f_mainline_commit}`.

Reply:
We should add a skipGodeps check in amend-godeps-at and also where godep-changes is called (or even in that func).

Reply:
that will require plumbing all the way down from sync-repo, not sure if it is worth for this case.